From 3c6727ae67ef3c756ab17b500a5bdfe49f5cadc2 Mon Sep 17 00:00:00 2001
From: Trent Mick <trentm@gmail.com>
Date: Tue, 27 Feb 2018 19:41:19 -0800
Subject: [PATCH] TRITON-193 TRITON-185 broke sapi startup: "Uncaught Error:
 ENOENT: no such file or directory, open
 '/opt/smartdc/sapi/lib/server/../../package.json'"

---
 Makefile | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Makefile b/Makefile
index 48c0103..5a37add 100644
--- a/Makefile
+++ b/Makefile
@@ -95,6 +95,7 @@ service: all $(SMF_MANIFESTS)
 	@touch $(SVC_PKGDIR)/site/.do-not-delete-me
 	cp -r $(TOP)/server.js \
 		$(TOP)/node_modules \
+		$(TOP)/package.json \
 		$(SVC_INSTDIR)/
 	cp -r $(TOP)/lib/common \
 		$(TOP)/lib/server \
-- 
2.21.0

